doc/user/content/sql/show-types.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "SHOW TYPES"
+description: "`SHOW TYPES` returns a list of the data types in your Materialize instance."
+menu:
+  main:
+    parent: 'sql'
+---
+
+`SHOW TYPES` returns a list of the data types in your Materialize instance. By default, only user-created columns are returned.
```

Comment:
I think this should read `By default, only user-create types are returned` instead of `columns`.